bin/lolcommits

```diff
@@ -408,10 +332,10 @@ if not (Choice.choices[:enable] || Choice.choices[:disable])
   elsif Choice.choices[:last]
     do_last
   elsif Choice.choices[:browse]
-    die_if_not_git_repo!
-    Launchy.open configuration.loldir
+    Fatals.die_if_not_git_repo!
+    Launcher.open_folder(configuration.loldir)
   elsif Choice.choices[:gif]
-    do_gif
+    HistoryAnimator.new(configuration).run(Choice.choices[:gif])
```

Comment:
Yeah I held off on doing anything with Choice since I want to factor it out entirely eventually, and the Configuration class is still a mess.  More notes on that coming in comment.